### PR TITLE
{bio}[GCC/11.2.0,gompi/2021b] EvidentialGene v2022.01.14, Exonerate v2.4.0

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/e/EvidentialGene/EvidentialGene-2022.01.14-gompi-2021b.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+sources = ['evigene%s.tar' % local_dlver]
+checksums = ['d464240affb6d293f20dab440abe89381e580154502e777c020c0060c8947ba3']
+
+dependencies = [
+    ('Perl', '5.34.0'),
+    ('Exonerate', '2.4.0'),
+    ('CD-HIT', '4.8.1'),
+    ('BLAST+', '2.12.0'),
+]
+
```
Comment (branfosj) [inline easybuild/easyconfigs/e/EvidentialGene/EvidentialGene-2022.01.14-gompi-2021b.eb:30]: The Perl scripts, in `scripts` and subdirectories, have `#!/usr/bin/perl`. So we need a `fix_perl_shebang_for = ...` to be run on them.

**Review by branfosj — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/e/EvidentialGene/EvidentialGene-2022.01.14-gompi-2021b.eb (hunk 1 above)
Comment (smoors) [reply easybuild/easyconfigs/e/EvidentialGene/EvidentialGene-2022.01.14-gompi-2021b.eb:30]: good catch, fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/15576/commits/73a0f7c6d717d9ff9abd2e00e46784e941eb5d69

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/5b99b79071e3ee6e5955918696dbbeef for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @smoors!

**Pull request merged**